src/idiomorph.js

```diff
@@ -365,12 +365,17 @@ var Idiomorph = (function () {
 
       // if we are at the end of the exiting parent's children, just append
       if (insertionPoint == null) {
-        // skip add callbacks when we're going to be restoring this from the pantry in the second pass
         if (
           ctx.config.twoPass &&
           ctx.persistentIds.has(/** @type {Element} */ (newChild).id)
```

Comment:
here if twoPass is false the persistentIds is an empty set so it may be possible to remove the twoPass config check here as well.  There could be a very small perf hit to check has on an empty set here but its probably not measurable either way.